evm/p2p/lightchain.py

```diff
@@ -93,7 +100,7 @@ def peer_block_height(peer):
             peer = await asyncio.wait_for(
                 handshake(remote, self.privkey, self.peer_class, self.chaindb, self.network_id),
                 HANDSHAKE_TIMEOUT)
-            return peer
+            return cast(LESPeer, peer)
```

Comment:
Is the `cast` call here because the `handshake` method takes the `peer_class` and since it's dynamic we have to hint at what type this is?

Reply:
Yep, although here I could have the `connect()` method returning a `BasePeer` as well and avoid the `cast`, but I think at this point it makes more sense to do this